setup.py

```diff
@@ -54,7 +54,8 @@
     if package == MODULE_DIR or package.startswith(MODULE_DIR + ".")
 ]
 install_requires = [
-    "urllib3>=1.26.18, <2",
+    'urllib3>=1.26.18,<1.27 ; python_version < "3.10"',
```

Comment:
Why `<1.27` rather than `<2`? Surely we want those who cannot use urllib3 2.x to at least get updates to 1.x?

Reply:
This also mirrors what `botocore` did. I believe the original rationale was that 1.27 may introduce some breaking changes.